src/store/antelope.store.ts

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable @typescript-eslint/no-unsafe-call */
+/* eslint-disable @typescript-eslint/no-unsafe-return */
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+
+import { StateInterface, useStore } from 'src/store';
+import { ResourcesAPI } from 'src/store/resources';
+import { DelegatedResources } from 'src/store/resources/state';
+import { Ref } from 'vue';
+import { DispatchOptions } from 'vuex';
+
+export interface AntelopeStore {
+    resources: ResourcesAPI;
+    state: StateInterface;
+    dispatch: (t: string, p?: unknown, o?: DispatchOptions) => Promise<unknown>;
```

Comment:
can we get more informative variable names here? rather than `t` `p` and `o`

Reply:
sure